CONTRIBUTING.md

```diff
@@ -1,121 +1,5 @@
 # Contributing to Semantic Kernel
```

Comment:
CONTRIBUTING.md is a very standard file to have in the root of a repo. 
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors
I'd like for this one (and README, ofc) to stay here. Moving the other files is fine.